shared/dto/widgets/page-filter.entity.ts

```diff
@@ -0,0 +1,14 @@
+import { CustomValueTransformers } from '@shared/lib/custom-value-transformers';
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+@Entity('page_filters')
+export class PageFilter {
+  @PrimaryColumn()
+  name: string;
+
+  @Column({
+    type: 'text',
+    transformer: CustomValueTransformers.semiColonSeparator,
```

Comment:
This part is evil AF. I am not sure if it's done this way because this is a temporal entity, but the transformer is being executed in each CRUD of this entity.

The actual algorithm is being executed in NodeJS thread, and it's a blocking operation. There is no huge amount of data to be manipulated, but filtering is a feature that will be executed the MOST